### PR TITLE
Move the Performance tab in settings page to user preferences page

### LINKED ISSUE #3891: [UI] Move the Performance tab in settings page to user preferences page
#### Current Behavior
In settings page, we have a tab called _Performance_ which is supposed to set the default configs for load test for that particular user. It makes more sense for it to be in  user preferences page since the load test configs are specific to a user and not to a system.

Screenshot 2021-08-15 at 7 31 17 PM

#### Desired Behavior
Performance tab in settings page should be completed moved to the user preferences page

Screenshot 2021-08-15 at 7 28 57 PM

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by ghost**
Signed-Off-by: Yash Agarwal <yagarwal1307@gmail.com>

This PR fixes #3891

### CODE REVIEW

File: ui/components/MesherySettings.js [hunk 1]
```diff
@@ -10,14 +10,13 @@ import Typography from '@material-ui/core/Typography';
...
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faArrowLeft, faCloud, faPoll, faTachometerAlt } from '@fortawesome/free-solid-svg-icons';
+import { faArrowLeft, faCloud, faPoll } from '@fortawesome/free-solid-svg-icons';
 import { faMendeley } from '@fortawesome/free-brands-svg-icons';
 import Link from 'next/link';
 import MeshConfigComponent from './MeshConfigComponent';
 import GrafanaComponent from './GrafanaComponent';
 import MeshAdapterConfigComponent from './MeshAdapterConfigComponent';
 import PrometheusComponent from './PrometheusComponent';
-import MesherySettingsPerformanceComponent from "../components/MesherySettingsPerformanceComponent";
```
Comment (leecalcote) [inline ui/components/MesherySettings.js:20]: @sudo-NithishKarthik I believe that we're going to want to retain this component on the /settings page, but have it invoke a different API endpoint than it has in the past.

**Comment by warunicorn19** [comment]
Also head up, merge conflicts

File: ui/components/MesherySettings.js (hunk 1 above)
Comment (sudo-NithishKarthik) [reply ui/components/MesherySettings.js:20]: Okay, so we will have it in both places ?

**Comment by sudo-NithishKarthik** [comment]
@yagarwal1307 , May I know why you closed this PR ?

**Comment by yagarwal1307** [comment]
@sudo-NithishKarthik actually I deleted the branch I was working on, in my remote repo mistakenly and this automatically closed the issue. So I had to create a new PR for the same.

**Pull request closed without merging**